templates/taxi/logout.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Logged Out</title>
+</head>
+<body>
+  <h1>You have been logged out</h1>
+  <p><a href="{% url 'login' %}">Log in again</a> or <a href="{% url 'home' %}">return to home page</a>.</p>

```

Comment:
The URL tag `{% url 'login' %}` should match the name of the login path defined in your `urls.py`. Ensure consistency between the template and URL configuration to avoid navigation issues.